Flank/flank
# feat: Add support for emulator devices

In the scope of #1710

PR adds support for `EMULATOR` devices

## Test Plan
> How do we know the code works?

1. build flank locally with `./gradlew flankFullRun`
2. ensure you are logged to `flank-open-source` project
3. run `flank firebase test android models list`
4. you should see new devices of type `EMULATOR`
5. run a simple test with
    ```
      device:
         - model: NexusLowResEmulator
           version: 28
    ```
6. tests should finish normally

## Checklist
